pyodata/v2/service.py

```diff
@@ -1177,10 +1177,17 @@ def __getattr__(self, name):
         def function_import_handler(fimport, response):
             """Get function call response from HTTP Response"""
 
-            if response.status_code != requests.codes.ok:
-                raise HttpError('Function import call failed with status code {0}'.format(response.status_code),
+            if response.status_code >= 300:
```

Comment:
Wouldn't it be better to handle `Redirects` interval [300, 399] separately and report that current transport layer (requests) doesn't support redirects?. Reporting result as `failed` could be misleading in some cases. And of course, leave current `failed` reporting for both client range [400-499] and server range >500.

Reply:
Thanks for the comment! I forgot I want to change error handling to something less stupid.

Reply:
I added one more commit which is supposed to make error reporting less confusing.